_maps/shuttles/whiteship_meta.dmm

```diff
@@ -677,13 +651,25 @@
 	dir = 1
 	},
 /area/shuttle/abandoned)
-"bm" = (
+"ba" = (
+/obj/machinery/porta_turret/aux_base{
```

Comment:
Make these a subtype of porta_turret instead of copypasting the same vareditted item everywhere.